### PR TITLE
Removed old Loc.GetString() use instances

### PR DESCRIPTION
**Pull request by GalacticChimp**
Requires https://github.com/space-wizards/space-station-14/pull/4155 - client & shared side changes

**WARNING!** They need to be merged at the same time (this one preferably second)!

### CODE REVIEW

**Comment by GalacticChimp** [comment]
Server build fails because it needs some stuff from Client's PR Shared parts

File: Robust.Client/Console/Commands/Scripting.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@ namespace Robust.Client.Console.Commands
     internal sealed class ScriptCommand : IConsoleCommand
     {
         public string Command => "csi";
-        public string Description => "Opens a C# interactive console.";
+        public string Description => Loc.GetString("console-script-command-description");
```
Comment (PJB3005) [inline Robust.Client/Console/Commands/Scripting.cs]: I'd name these something like `concmd-Script-desc`.

Though I'm also not sure we should really bother localizing console commands?

**Review by PJB3005 — COMMENTED** [review]
Move localization strings that are used from engine into the engine, please. 

(I'm assuming you have them in content right now)

File: Robust.Client/Console/Commands/Scripting.cs (hunk 1 above)
Comment (GalacticChimp) [reply Robust.Client/Console/Commands/Scripting.cs]: I can revert console command changes, I simply wasn't sure if we needed them

Comment (GalacticChimp) [reply Robust.Client/Console/Commands/Scripting.cs]: ok, removed Loc.GetString() from console commands, they didn't make sense

**Comment by GalacticChimp** [comment]
ok, I think everything is fixed now

**Pull request merged**